core/resources/resources.go

```diff
@@ -12,13 +12,13 @@ import (
 // DockerImageDetails holds the details for a Docker resource type.
 type DockerImageDetails struct {
 	// RegistryPath holds the path of the Docker image (including host and sha256) in a docker registry.
-	RegistryPath string
+	RegistryPath string `json:"ImagePath"`
```

Comment:
the json tags are CamelCase not lower case?

Reply:
yes.